client/my-sites/checkout/src/lib/translate-payment-method-names.ts

```diff
@@ -1,6 +1,11 @@
+import config from '@automattic/calypso-config';
 import { camelToSnakeCase } from '@automattic/js-utils';
 import type { CheckoutPaymentMethodSlug, WPCOMPaymentMethod } from '@automattic/wpcom-checkout';
 
+const StripeRedirectMigrationBancontact = config.isEnabled(
```

Comment:
I see it was mentioned in the other PR but I would call this something like `isStripeRedirectMigrationBancontact` (lowercase first letter) to show that it's not a class or component. Lowercase first letters for variables are not a language requirement but a very common JavaScript pattern.